ozaria/engine/cinematic/CinematicLankBoss.js

```diff
@@ -17,9 +17,9 @@ import {
   getSpeaker,
   getHeroPet
 } from '../../../app/schemas/models/selectors/cinematic'
-import { LETTER_ANIMATE_TIME, getHeroSlug } from './constants'
+import { LETTER_ANIMATE_TIME } from './constants'
 
-export const HERO_THANG_ID = '55527eb0b8abf4ba1fe9a107'
+export const HERO_THANG_ID = '5d03e18887ed53004682e340'
```

Comment:
Should this be moved to constants?

Reply:
Absolutely it should be. Thanks for the comment.